api/envoy/config/core/v3/substitution_format_string.proto

```diff
@@ -41,11 +43,11 @@ message SubstitutionFormatString {
     // Nested JSON objects may be produced by some command operators (e.g. FILTER_STATE or DYNAMIC_METADATA).
     // See the documentation for a specific command operator for details.
     //
-    // .. code-block::
+    // .. code-block:: yaml
```

Comment:
Maybe consider using https://github.com/envoyproxy/envoy/pull/11394 if it works here.

Reply:
Sorry, should the config validation be done automatically?

Reply:
From looking at the PR, you would need to switch to `validated-code-block`.

Reply:
Thanks! Updated. But yeah, for `$REQ()%` vs `%REQ()%` I wish we can catch that too.